templates/cli/lib/parser.js.twig

```diff
@@ -110,12 +111,18 @@ const drawJSON = (data) => {
 }
 
 const parseError = (err) => {
+    if(cliConfig.report) {
+        const githubIssueUrl=`https://github.com/appwrite/appwrite/issues/new?labels=bug&template=bug.yaml&title=%F0%9F%90%9B+Bug+Report%3A+&actual-behavior=Console%20Crash%0A${encodeURIComponent('```\n'+err.stack+'\n```').replaceAll('(','').replaceAll(')','')}`
```

Comment:
Amazing beginning 🔥 Let's add "wow" effect to it:

- In `steps to reproduce`, can we include command that developer ran?
- In `your environment`, can we include CLI version?
- Can we make `operating system` auto-select correct system? CI should know
- For `appwrite version`, let's select cloud if endpoint is cloud, and "custom version" if self-hosted. If self-hosted, lets specify appwrite server version in "your environment".
- Lets prefill title with `🐛 Bug Report: CLI error: [ERROR_MESSAGE_HERE]`